runtime/rastrace/method_trace.c

```diff
@@ -486,13 +486,16 @@ traceMethodArguments(J9VMThread* thr, J9UTF8* signature, UDATA* arg0EA, char* bu
 
 	while (*(++sigChar) != ')') {
 		switch (*sigChar) {
-		case '[':
+		case '[': /* fall through */
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+		case 'Q': /* fall through */
+#endif /* #if defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
 		case 'L':
 			traceMethodArgObject(thr, arg0EA--, cursor, endOfBuf - cursor);
 			while (*sigChar == '[') {
 				sigChar++;
 			}
-			if (*sigChar == 'L' ) {
+			if (*sigChar == 'L' || *sigChar == 'Q') {
```

Comment:
Is there a reason why `IS_REF_OR_VAL_SIGNATURE()` is not used here ?

Reply:
Nope, I just forgot, changed it in f180a75